inspire-tree.d.ts

```diff
@@ -150,6 +150,8 @@ export class InspireTree extends EventEmitter2 {
     hide(): TreeNodes;
     hideDeep(): TreeNodes;
     id: string | number;
+    config: any;
```

Comment:
Neither of these were intended to be public/advertised but since they do exist (and have no alternatives), I should include them here.

This config property should be a type of `Config`.

Reply:
Updated to use config.

Also added a few other missing properties.
I have been implementing a custom renderer, which touches these properties. (and per the docs is a possible use case)